payloads/external/edk2/Makefile

```diff
@@ -253,6 +253,9 @@ prep: $(EDK2_PATH) clean checktools logo lan_rom ipxe_rom
 $(WORKSPACE)/Build/UefiPayloadPkgX64/$(RELEASE_STR)_COREBOOT/FV/UEFIPAYLOAD.fd: \
 	prep print
 	cd $(WORKSPACE); \
+		echo -n "Building iPXE... "; \
+		cd $(PACKAGES_PATH)/UefiPayloadPkg/NetworkDrivers; \
+		./build-ipxe.sh; \
```

Comment:
I just ditched this and enabled building iPXE through coreboot build system. Please do not include this commit.